### PR TITLE
[docs] Changelog for v0.21.0

### PR DESCRIPTION
**Pull request by VasilyStrelyaev**
\cc @DevExpress/testcafe-docs @miherlosev 

Changelog for v0.21.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,91 @@
...
+            .browsers('chrome --allow-insecure-localhost')
+            .run();
+    });
+```
+
+See [Connect to TestCafe Server over HTTPS](https://devexpress.github.io/testcafe/documentation/using-testcafe/common-concepts/connect-to-the-testcafe-server-over-https.html) for more information.
+
+#### :gear: Construct Screenshot Paths with Patterns ([#2152](https://github.com/DevExpress/testcafe/issues/2152))
+
+You can now use patterns to construct paths to screenshots. TestCafe provides a number of variables you can include in the path. For instance, these are `${DATE}`, `${TIME}`, `${USERAGENT}`, etc. For a complete list, refer to the command line [--screenshot-path-pattern flag description](https://devexpress.github.io/testcafe/documentation/using-testcafe/command-line-interface.html#-p---screenshot-path-pattern).
```
Comment (miherlosev) [inline CHANGELOG.md]: Instead of `variable` word use the `placeholder`.

'TestCafe provides a number of placeholders...'

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,91 @@
...
+
+You specify a screenshot path pattern when you run tests. Each time TestCafe takes a screenshot, it substitutes the variables with their values and saves the screenshot to the resulting path.
```
Comment (miherlosev) [inline CHANGELOG.md]: This phrase is not clear.
Something like this `..it substitutes the placeholders with actual values and saves the screenshot to ...`

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,91 @@
...
+
+#### :gear: Add Info About Screenshots and Quarantine Attempts to Custom Reports ([#2216](https://github.com/DevExpress/testcafe/issues/2216))
+
+Custom reporters can now access data about screenshots and the history of quarantine attempts (if the test ran in the quarantine mode).
+
+The following info about screenshots is now available:
+
+* the path to the screenshot file,
+* the path to the thumbnail image,
+* the user agent in which the screenshot was taken,
```
Comment (miherlosev) [inline CHANGELOG.md]: I remember that we discuss about it, but we cannot take screenshot in the user agent.
This should be more clear
```js
the user agent of the browser in which the screenshot was taken
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,92 @@
...
+* the quarantine attempt number (if the screenshot was taken in the quarantine mode),
+* whether the screenshot was taken because the test failed.
+
+If the test was run in the quarantine mode, you can also determine which attempts failed and passed.
+
+Refer to the [reportTestDone method description](https://devexpress.github.io/testcafe/documentation/extending-testcafe/reporter-plugin/reporter-methods.html#reporttestdone) for details on how to access this information.
+
+### Bug Fixes
+
+* The Drag action now works correctly in specific cases ([#2529](https://github.com/DevExpress/testcafe/issues/2529))
```
[the remark this answers is not in the document]
Comment (VasilyStrelyaev) [reply CHANGELOG.md]: Got more details from the developers. I think we can write it this way:

`* HTML5 drag events are no longer simulated if event.preventDefault is called for the mousedown event ([#2529](https://github.com/DevExpress/testcafe/issues/2529))`

[the remark this answers is not in the document]
Comment (dirk-pieterse) [reply CHANGELOG.md]: Ok

**Pull request merged**